modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -1103,6 +1105,19 @@ public void incrementViewCount(
 			increment);
 	}
 
+	public Lock lockKBArticle(long userId, long resourcePrimKey)
```

Comment:
Aren't you missing an `@Override` annotation here?

Reply:
Fixing